agora/Agora/Stake/Scripts.hs

```diff
@@ -133,7 +132,7 @@ import Prelude hiding (Num ((+)))
 -}
 stakePolicy ::
   -- | The (governance) token that a Stake can store.
-  ClosedTerm (PAssetClass :--> PMintingPolicy)
+  ClosedTerm (PAsData PAssetClassData :--> PMintingPolicy)
```

Comment:
don't need `PAsData` here

Reply:
`ptoScottEncoding::Term s (PAsData PAssetClassData :--> PAssetClass)`. I will open a pr to address this on LPE side.